test/test_tensordict.py

```diff
@@ -64,6 +65,27 @@ def test_tensordict_set(device):
     assert td._tensordict_meta["key1"].shape == td._tensordict["key1"].shape
 
 
+def test_pad():
```

Comment:
I left this in because I wanted to test that pre and post padding for each dimension was working properly. I could move this to the TestTensorDicts function but it seemed repetitive

Reply:
oh i see what you mean
ok that's perfect